pkg/kudoctl/files/file.go

```diff
@@ -88,3 +88,9 @@ func Sha256Sum(r io.Reader) (string, error) {
 	}
 	return fmt.Sprintf("%x", h.Sum(nil)), nil
 }
+
+// Exists returns true if the path exists
+func Exists(fs afero.Fs, path string) bool {
+	exists, _ := afero.Exists(fs, path)
```

Comment:
do you feel like such method makes sense? It converts one oneliner to another. Basically instead of calling something on afero we're passing it as parameter and then calling it on afero...

Reply:
I'm a little worried about this too because it's swallowing the error. For example, this would swallow a permissions error - afero is only swallowing the os.IsNotExist error.